src/components/Body/index.jsx

```diff
@@ -1,61 +1,72 @@
 
 const Body = () => {
+  const settings = {
+    dots: false,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 3,
+    slidesToScroll: 3,
+  };
   return (
-    <Grid container spacing={3} marginTop={5} marginBottom={13}>
-      {/* Main Content Sections */}
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-        <Typography variant="h1" gutterBottom textAlign={"center"}>
-          New Reviews
-        </Typography>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-        <Grid container spacing={2} xs={12}>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-        </Grid>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-    </Grid>
+    // <Grid container spacing={3} marginTop={5} marginBottom={13}>
+    //   {/* Main Content Sections */}
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}>
+    //     <Typography variant="h1" gutterBottom textAlign={"center"}>
+    //       New Posts
+    //     </Typography>
+    //   </Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}>
+    //     <Grid container spacing={2} xs={12}>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //     </Grid>
+    //   </Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}></Grid>
+    //   {/* <Grid item xs={4} md={4} lg={2}></Grid> */}
+    // </Grid>
+    <>
+      <h1 className="body-post">New Posts</h1>
```

Comment:
The text should be New Reviews

Reply:
untuk ini saya menyesuaikan dengan design figma

Reply:
iy, yg di figma ga sesuai. Endpoint nya juga sudah dibuat namanya review, bukan Post